DEPS

```diff
@@ -151,6 +151,9 @@ deps = {
 
   # WARNING: Unused Dart dependencies in the list below till "WARNING:" marker are removed automatically - see create_updated_flutter_deps.py.
 
+  'src/third_party/dart/third_party/devtools':
+   {'packages': [{'version': 'git_revision:12ad5341ae0a275042c84a4e7be9a6c98db65612', 'package': 'dart/third_party/flutter/devtools'}], 'dep_type': 'cipd'},
```

Comment:
I don't think so. Every time I put in something different than this the 3xH script adds this line and then complains. I think the DEPS update script automatically updates these values in place instead of using the Vars.